webservice/getting-started.md

```diff
@@ -192,7 +193,7 @@ This parameter can only be used for listings, not for individual records. If you
 {{% notice note %}}
 A response obtained with "display" other than "full" can't be used in a **PUT** (update) request, because the `WebserviceRequest` class validation for fields is the same for **POST** (create) and **PUT** (update).
 
-This should be fixed in a near future with a *yet-to-come-pull-request* introducing the PATCH method!
+**PATCH** method allow you to use a partial display !
```

Comment:
```suggestion
The **PATCH** method allows you to use a partial display.
```

Also, doesn't this statement cancel the previous one? Then we should remove it too, right ?

Reply:
@PrestaEdit you forgot about this one

Reply:
@kpodemski No, I was just away from keyboard between the two suggestions changes ;)